### PR TITLE
plasm-infra: Split lint and unit tests to different jobs 

### PR DESCRIPTION
**Pull request by Yakutoc**
## Release Notes

Разделили первичные и важные проверки, такие как Lint и Unit tests,  в рамках одного workflow на две разные jobs

## What Changed

Для атомарности.  

**Unit tests** запускается с помощью `lerna run test` и поэтому мы можем использовать флаг **`--since`** и для `lerna bootstrap`, и для самой команды запуска тестов. 

А вот **Lint** запускается для всей монорепы, и так получается что флаг `--since` уже мешает и вносит проблемы, в виде неустановленных пакетов. 

Поэтому для этой проверки оставили **полный** bootstrap.   

## Check list 

1. plasma-core
2. plasma-web
3. plasma-tokens-utils          

Ни в одном сценарии **не было** ошибки связной с неустановленным/отсутствующим пакетом.    

https://github.com/salute-developers/plasma/actions/workflows/tests.yml?query=branch%3Aci_refactoring_tests-workflow

--------- 

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.631.5690535609.swift'>color_plasma_b2c_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.631.5690535609.kt'>color_plasma_b2c_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.631.5690535609.ts'>color_plasma_b2c_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.631.5690535609.xml'>color_plasma_b2c_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.631.5690535609.swift'>color_sberHealth_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.631.5690535609.kt'>color_sberHealth_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.631.5690535609.ts'>color_sberHealth_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.631.5690535609.xml'>color_sberHealth_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.631.5690535609.swift'>color_sbermarket_business_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.631.5690535609.kt'>color_sbermarket_business_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.631.5690535609.ts'>color_sbermarket_business_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.631.5690535609.xml'>color_sbermarket_business_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.631.5690535609.swift'>color_sbermarket_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.631.5690535609.kt'>color_sbermarket_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.631.5690535609.swift'>color_sbermarket_metro_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.631.5690535609.kt'>color_sbermarket_metro_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.631.5690535609.ts'>color_sbermarket_metro_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.631.5690535609.xml'>color_sbermarket_metro_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.631.5690535609.ts'>color_sbermarket_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.631.5690535609.swift'>color_sbermarket_selgros_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.631.5690535609.kt'>color_sbermarket_selgros_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.631.5690535609.ts'>color_sbermarket_selgros_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.631.5690535609.xml'>color_sbermarket_selgros_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.631.5690535609.swift'>color_sbermarket_wlbusiness_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.631.5690535609.kt'>color_sbermarket_wlbusiness_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.631.5690535609.ts'>color_sbermarket_wlbusiness_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.631.5690535609.xml'>color_sbermarket_wlbusiness_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.631.5690535609.xml'>color_sbermarket_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.631.5690535609.swift'>color_sberonline_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.631.5690535609.kt'>color_sberonline_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.631.5690535609.ts'>color_sberonline_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.631.5690535609.xml'>color_sberonline_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.631.5690535609.swift'>color_sberprime_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.631.5690535609.kt'>color_sberprime_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.631.5690535609.ts'>color_sberprime_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.631.5690535609.xml'>color_sberprime_xml--canary.631.5690535609.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.631.5690535609.swift'>color_stylesSalute_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.631.5690535609.kt'>color_stylesSalute_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.631.5690535609.ts'>color_stylesSalute_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.631.5690535609.xml'>color_stylesSalute_xml--canary.631.5690535609.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.631.5690535609.ts'>shadow_plasma_b2c_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.631.5690535609.ts'>shadow_sberHealth_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.631.5690535609.ts'>shadow_sbermarket_business_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.631.5690535609.ts'>shadow_sbermarket_metro_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.631.5690535609.ts'>shadow_sbermarket_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.631.5690535609.ts'>shadow_sbermarket_selgros_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.631.5690535609.ts'>shadow_sbermarket_wlbusiness_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.631.5690535609.ts'>shadow_sberonline_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.631.5690535609.ts'>shadow_sberprime_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.631.5690535609.ts'>shadow_stylesSalute_react-native--canary.631.5690535609.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.631.5690535609.swift'>typo_mage_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.631.5690535609.kt'>typo_mage_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.631.5690535609.ts'>typo_mage_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.631.5690535609.swift'>typo_plasma_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.631.5690535609.kt'>typo_plasma_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.631.5690535609.ts'>typo_plasma_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.631.5690535609.swift'>typo_ruler_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.631.5690535609.kt'>typo_ruler_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.631.5690535609.ts'>typo_ruler_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.631.5690535609.swift'>typo_sage_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.631.5690535609.kt'>typo_sage_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.631.5690535609.ts'>typo_sage_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.631.5690535609.swift'>typo_sbermarket_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.631.5690535609.kt'>typo_sbermarket_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.631.5690535609.ts'>typo_sbermarket_react-native--canary.631.5690535609.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.631.5690535609.swift'>typo_soulmate_ios-swift--canary.631.5690535609.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.631.5690535609.kt'>typo_soulmate_kotlin--canary.631.5690535609.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.631.5690535609.ts'>typo_soulmate_react-native--canary.631.5690535609.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.219.1-canary.631.5690535609.0
  npm install @salutejs/plasma-core@1.124.1-canary.631.5690535609.0
  npm install @salutejs/plasma-hope@1.219.1-canary.631.5690535609.0
  npm install @salutejs/plasma-icons@1.153.1-canary.631.5690535609.0
  npm install @salutejs/plasma-temple@1.168.1-canary.631.5690535609.0
  npm install @salutejs/plasma-tokens-b2b@1.23.1-canary.631.5690535609.0
  npm install @salutejs/plasma-tokens-b2c@0.32.1-canary.631.5690535609.0
  npm install @salutejs/plasma-tokens-web@1.38.1-canary.631.5690535609.0
  npm install @salutejs/plasma-tokens@1.57.1-canary.631.5690535609.0
  npm install @salutejs/plasma-ui@1.200.1-canary.631.5690535609.0
  npm install @salutejs/plasma-web@1.219.1-canary.631.5690535609.0
  npm install @salutejs/plasma-cy-utils@0.64.1-canary.631.5690535609.0
  npm install @salutejs/plasma-sb-utils@0.122.1-canary.631.5690535609.0
  npm install @salutejs/plasma-tokens-utils@0.28.1-canary.631.5690535609.0
  # or 
  yarn add @salutejs/plasma-b2c@1.219.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-core@1.124.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-hope@1.219.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-icons@1.153.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-temple@1.168.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-tokens-b2b@1.23.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-tokens-b2c@0.32.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-tokens-web@1.38.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-tokens@1.57.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-ui@1.200.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-web@1.219.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-cy-utils@0.64.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-sb-utils@0.122.1-canary.631.5690535609.0
  yarn add @salutejs/plasma-tokens-utils@0.28.1-canary.631.5690535609.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-631/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-631/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-631/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-631/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-631/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-631/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-631/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-631/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-631/web-storybook/
ui storybook: http://plasma.sberdevices.ru/pr/pr-631/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-631/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-631/
web storybook: http://plasma.sberdevices.ru/pr/pr-631/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-631/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-631/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-631/web-storybook/
ui storybook: http://plasma.sberdevices.ru/pr/pr-631/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-631/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

File: .github/workflows/required-primary-checks.yml [hunk 1]
```diff
@@ -12,33 +12,50 @@ concurrency:
...
+    name: Change detection
     uses: ./.github/workflows/change-detection.yml
     secrets: inherit
 
-  build:
-    needs: scope
-    name: Build
-    if: ${{ needs.scope.outputs.HAS_SCOPE == 'true' }}
+  lint:
+    needs: change-detection
```
Comment (neretin-trike) [inline .github/workflows/required-primary-checks.yml:27]: а здесь точно эта проверка нужна? Просто мы в итоге в конце выполняем просто `npx lerna bootstrap`

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-631/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-631/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-631/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-631/web-storybook/

File: .github/workflows/required-primary-checks.yml [hunk 2]
```diff
@@ -12,33 +12,41 @@ concurrency:
...
     steps:
       - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
 
       - uses: ./.github/actions/prepare-environment
 
       - name: Lerna bootstrap
-        run: npx lerna bootstrap --ignore @salutejs/plasma-tokens*
+        run: npx lerna bootstrap
```
Comment (Yeti-or) [inline .github/workflows/required-primary-checks.yml:31]: я же правильно помню что потом мы разделим линтер по пакетам и втащим `--since` ?

**Review by Yeti-or — APPROVED** [review]

File: .github/workflows/required-primary-checks.yml (hunk 2 above)
Comment (Yakutoc) [reply .github/workflows/required-primary-checks.yml:31]: @Yeti-or Да все правильно, у нас есть задача на это.

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.220.0, @salutejs/plasma-core@1.125.0, @salutejs/plasma-cy-utils@0.65.0, @salutejs/plasma-hope@1.220.0, @salutejs/plasma-icons@1.154.0, @salutejs/plasma-sb-utils@0.123.0, @salutejs/plasma-temple-docs@0.160.0, @salutejs/plasma-temple@1.169.0, @salutejs/plasma-tokens-b2b@1.24.0, @salutejs/plasma-tokens-b2c@0.33.0, @salutejs/plasma-tokens-native@1.9.0, @salutejs/plasma-tokens-utils@0.29.0, @salutejs/plasma-tokens-web@1.39.0, @salutejs/plasma-tokens@1.58.0, @salutejs/plasma-typo@0.36.0, @salutejs/plasma-ui-docs@0.209.0, @salutejs/plasma-ui@1.201.0, @salutejs/plasma-web-docs@0.178.0, @salutejs/plasma-web@1.220.0, @salutejs/plasma-website@0.186.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.220.0, @salutejs/plasma-core@1.125.0, @salutejs/plasma-cy-utils@0.65.0, @salutejs/plasma-hope@1.220.0, @salutejs/plasma-icons@1.154.0, @salutejs/plasma-sb-utils@0.123.0, @salutejs/plasma-temple-docs@0.160.0, @salutejs/plasma-temple@1.169.0, @salutejs/plasma-tokens-b2b@1.24.0, @salutejs/plasma-tokens-b2c@0.33.0, @salutejs/plasma-tokens-native@1.9.0, @salutejs/plasma-tokens-utils@0.29.0, @salutejs/plasma-tokens-web@1.39.0, @salutejs/plasma-tokens@1.58.0, @salutejs/plasma-typo@0.36.0, @salutejs/plasma-ui-docs@0.209.0, @salutejs/plasma-ui@1.201.0, @salutejs/plasma-web-docs@0.178.0, @salutejs/plasma-web@1.220.0, @salutejs/plasma-website@0.186.0 🚀

**Pull request merged**